infrastructure/kubernetes/monitoring/grafana/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: grafana
+  namespace: monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      labels:
+        app: grafana
+    spec:
+      containers:
+        - name: grafana
+          image: grafana/grafana:latest
+          ports:
+            - containerPort: 3000
+          env:
+            - name: GF_SECURITY_ADMIN_PASSWORD
+              value: "admin"  # TODO: change this
```

Comment:
We need to use secrets for that. Let's assume it's loaded from an existing secret and mention the secret creation step in the `monitoring/README.md` (we need to add it anyway to describe the monitoring stack).

Reply:
addressed in: 2bc29ab68c99605cf86930243bc236d1ebe02987